tests/docker/Dockerfile.install

```diff
@@ -45,7 +45,7 @@ RUN cd /comdb2 &&  \
     rm -rf build && \
     mkdir build && \
     cd build && \
-    cmake -DWITH_TCL=1 .. && \
+    cmake -DCMAKE_BUILD_TYPE=Debug -DWITH_TCL=1 .. && \
```

Comment:
Will this run tests for roborivers with Debug on? If so, I may be wrong but roboperf uses that same executable, so we will start seeing different performance numbers -- it may not be a good idea to enable here for now.

Reply:
This change (when I make it) will always be manually backed out prior to merge.